src/main/java/org/jabref/gui/fieldeditors/contextmenu/ClearField.java

```diff
@@ -0,0 +1,18 @@
+package org.jabref.gui.fieldeditors.contextmenu;
+
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.TextArea;
+
+import org.jabref.logic.formatter.bibtexfields.NormalizeNamesFormatter;
+import org.jabref.logic.l10n.Localization;
+
+class ClearField extends MenuItem {
+
+    private final TextArea opener;
+
+    public ClearField(TextArea opener) {
+        super(Localization.lang("Clear text"));
+        this.opener = opener;
+        setOnAction(event -> opener.setText(new NormalizeNamesFormatter().format("")));
```

Comment:
Why do you use the names formatter here? I think `TextArea` has a `clear` method, or otherwise `setText("")` should work.

Reply:
Hi, I was on the team with Weida that was working on this issue. We now realize that the things you mention were not needed. We were trying to follow how the other menu item classes were built, and that is why we added the private field. We found the class to add the menu item to the menuItem array by finding a similar issue that dealt with the Normalize name option. But we see now that we made it much more complicated than it needed to be. We should be able to fix these this week if we need to. 
Thanks